ui/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, Typography } from '@mui/material';
+import CopyrightIcon from '@mui/icons-material/Copyright';
+
+export default function Footer() {
+    return (
+        <Box sx={{
+            textAlign: 'center',
+            padding: 2,
+            backgroundColor: '#1a1a1a',
```

Comment:
Colors should be globlar instead of providing sperate values as such for later implementation of changing page style.